test/base/TextTest.ooc

```diff
@@ -166,6 +166,13 @@ TextTest: class extends Fixture {
 			text free()
 			text2 free()
 		})
+		this add("trim", func {
+			paddedText := t"  \t test \n test \r\n\t "
+			trimmedText := paddedText trim()
+			expect(trimmedText, is equal to(t"test \n test"))
```

Comment:
Having the whole test crash if something's wrong with `Text` isn't very good :) So yes, until my war against the SDK has been won.